lib/android/app/src/main/java/com/reactnativenavigation/views/stack/topbar/titlebar/TitleComponentMeasurer.kt

```diff
@@ -0,0 +1,32 @@
+package com.reactnativenavigation.views.stack.topbar.titlebar
+
+import kotlin.math.max
+
+fun resolveTitleBoundsLimit(
+        defaultStartMargin: Int,
+        parentWidth: Int,
+        titleWidth: Int,
+        leftBarWidth: Int,
+        rightBarWidth: Int,
+        isCenter: Boolean
+): Pair<Int, Int> {
```

Comment:
Perhaps consider changing the return type to:

```kotlin
typealias RightLimit = Int
typealias StartX = Int

Pair<StartX, RightLimit>
```

To make the code more readable.

Reply:
Awesome!